<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>theme</class>
 <widget class="QDialog" name="theme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>537</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>设置主题</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="statusTip">
      <string>主题选择器</string>
     </property>
     <property name="whatsThis">
      <string>主题选择器</string>
     </property>
     <property name="title">
      <string>主题选择</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0">
       <widget class="QLabel" name="theme_instruction">
        <property name="statusTip">
         <string>若要应用主题，请点击「应用」，您重启程序以后新的设置才会生效。</string>
        </property>
        <property name="whatsThis">
         <string>若要应用主题，请点击「应用」，您重启程序以后新的设置才会生效。</string>
        </property>
        <property name="text">
         <string>若要应用主题，请点击「应用」，您重启程序以后新的设置才会生效。</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="setTheme_Black">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="statusTip">
         <string>设置黑色主题</string>
        </property>
        <property name="whatsThis">
         <string>设置黑色主题</string>
        </property>
        <property name="text">
         <string>黑色</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="setTheme_Dark">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="statusTip">
         <string>设置深色主题</string>
        </property>
        <property name="whatsThis">
         <string>设置深色主题</string>
        </property>
        <property name="text">
         <string>深色</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="setTheme_light">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="statusTip">
         <string>设置浅色主题</string>
        </property>
        <property name="whatsThis">
         <string>设置浅色主题</string>
        </property>
        <property name="text">
         <string>浅色</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="statusTip">
         <string>自定义颜色框</string>
        </property>
        <property name="whatsThis">
         <string>自定义颜色框</string>
        </property>
        <property name="accessibleName">
         <string/>
        </property>
        <property name="title">
         <string>自定义</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="primary_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>CrossCursor</cursorShape>
           </property>
           <property name="statusTip">
            <string>自定义主色</string>
           </property>
           <property name="whatsThis">
            <string>自定义主色</string>
           </property>
           <property name="placeholderText">
            <string>主色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="background_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>CrossCursor</cursorShape>
           </property>
           <property name="statusTip">
            <string>自定义背景色</string>
           </property>
           <property name="whatsThis">
            <string>自定义背景色</string>
           </property>
           <property name="placeholderText">
            <string>背景色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="text_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>CrossCursor</cursorShape>
           </property>
           <property name="statusTip">
            <string>自定义被选中文字色</string>
           </property>
           <property name="whatsThis">
            <string>自定义被选中文字色</string>
           </property>
           <property name="placeholderText">
            <string>被选中文字色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="toolbar_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>CrossCursor</cursorShape>
           </property>
           <property name="statusTip">
            <string>自定义工具栏色</string>
           </property>
           <property name="whatsThis">
            <string>自定义工具栏色</string>
           </property>
           <property name="placeholderText">
            <string>工具栏色</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>44</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="apply_theme">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="statusTip">
         <string>应用主题</string>
        </property>
        <property name="whatsThis">
         <string>应用主题</string>
        </property>
        <property name="text">
         <string>应用</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="setTheme_Custom">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="statusTip">
         <string>设置自定义主题</string>
        </property>
        <property name="whatsThis">
         <string>设置自定义主题</string>
        </property>
        <property name="text">
         <string>自定义</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
